docs/pyproject.md

```diff
@@ -737,7 +738,17 @@ Here, we will have the `my_package_cli` script installed which will execute the
 When a script is added or updated, run `poetry install` to make them available in the project's virtualenv.
 {{% /note %}}
 
-### extras
+```toml
+[tool.poetry.scripts]
+my_executable = { reference = "some_binary.exe", type = "file" }
+```
+
+With this configuration, Poetry will look for the referenced file in the active project and then copy it to the OS installation directory.
```

Comment:
```suggestion
This tells Poetry to include the specified file, relative to your project directory, in distribution builds. It will then be copied to the appropriate installation directory for your operating system when your package is installed. 
```